test/support/utils.h

```diff
@@ -1243,6 +1243,47 @@ struct Pow2
     }
 };
 
+template <typename _T>
+struct NoDefaultCtorWrapper {
+    _T value;
+
+    // Default constructor
+    NoDefaultCtorWrapper() = delete;
```

Comment:
I think as far as we have another constructors here, this default deleted constructor doesn't make sense.

Reply:
I understand it is not necessary to write out, but since the point of this class is to delete the default, I think it is worthwhile to write it explicit for clarity